### PR TITLE
chore: ci updates and style change

### PR DESCRIPTION
**Pull request by Fdawgs**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
   "description": "A plugin for Fastify to enable management of cache control headers",
   "main": "plugin.js",
   "scripts": {
-    "test": "tap test/*.test.js",
-    "test-ci": "tap --cov test/*.test.js",
     "lint": "standard --verbose | snazzy",
-    "lint-ci": "standard"
+    "lint:ci": "standard",
+    "test": "tap test/*.test.js",
+    "test:ci": "tap --cov test/*.test.js"
```
Comment (mcollina) [inline package.json:10]: `test` and `test:ci` are identical, they both run the coverage.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Fdawgs) [reply package.json:10]: Glad you raised that, wanted to ask about it! Will review.

Comment (Fdawgs) [reply package.json]: Resolved with https://github.com/fastify/fastify-caching/pull/55/commits/d76e46ee2ed0fe87b2fe72ad1ca03a246a8b07b3

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**